src/wordlist.h

```diff
@@ -20,6 +42,7 @@ class wordlist
 {
     MEMPROXY_CLASS(wordlist);
     friend char *wordlistChopHead(wordlist **);
+    using iterator = WordlistIterator;
```

Comment:
It this needed? it appears unused. Or perhapse the `begin()` and `end()` methods should return `iterator` instead.